datajunction-server/datajunction_server/utils.py

```diff
@@ -52,12 +52,13 @@ def get_settings() -> Settings:
     return Settings()
 
 
+@lru_cache(maxsize=None)
 def get_engine() -> Engine:
     """
     Create the metadata engine.
     """
     settings = get_settings()
-    engine = create_engine(settings.index)
+    engine = create_engine(settings.index, echo=True)
```

Comment:
Yep! Although I'm going to remove this before merging. Might be nice to push this up to an server config setting so that we can quickly enable it if we need to debug, although I think having the trace functionality integrated also solves that problem.

Reply:
Which "trace functionality" ?

Reply:
The opentelemetry sqlalchemy instrumentor will let us trace database queries. We only have that implemented internally though.